test/e2e/run.sh

```diff
@@ -22,3 +25,19 @@ done
 # minimizes race condition in filebeat between reading log file and
 # stopping reading due to pod termination autodiscover event
 sleep 1
+}
+
+run_chaos() {
+  go run test/e2e/cmd/main.go chaos "$@"
+}
+
+
+main() {
+  if [ "${chaos}" = true ] ; then
```

Comment:
nit:
```suggestion
  if [ "${chaos}" == true ] ; then
```

Reply:
My understanding was that within a single bracket `=` and `==` behaves exactly the same. Any reason to prefer `==` ?

Reply:
Nit: many shell style guides recommend using double brackets `[[` which notably allows the use of classic comparison operators (&&, ||, <, >, !=, ==) (like this [one](https://google.github.io/styleguide/shellguide.html#test----and---)). I got into this habit but both suit me.